src/containers/Tenant/Diagnostics/TenantOverview/TenantOverview.tsx

```diff
@@ -0,0 +1,156 @@
+import {TABLET_STATES, TENANT_DEFAULT_TITLE} from '../../../../utils/constants';
+import {bytesToGB} from '../../../../utils/utils';
+import {mapDatabaseTypeToDBName} from '../../utils/schema';
+import {useAutofetcher, useTypedSelector} from '../../../../utils/hooks';
+import {getTenantInfo, setDataWasNotLoaded} from '../../../../store/reducers/tenant/tenant';
+
+import './TenantOverview.scss';
+
+const b = cn('tenant-overview');
+
+interface TenantOverviewProps {
+    tenantName: string;
+    additionalTenantInfo?: any;
+}
+
+export function TenantOverview({tenantName, additionalTenantInfo}: TenantOverviewProps) {
+    const {tenant, loading, wasLoaded} = useTypedSelector((state) => state.tenant);
+    const {autorefresh} = useTypedSelector((state) => state.schema);
+    const dispatch = useDispatch();
+    const fetchTenant = useCallback(
+        (isBackground = true) => {
+            if (!isBackground) {
+                dispatch(setDataWasNotLoaded());
+            }
+            dispatch(getTenantInfo({path: tenantName}));
+        },
+        [dispatch, tenantName],
+    );
+
+    useAutofetcher(
+        (isBackground) => {
+            fetchTenant(isBackground);
+        },
+        [fetchTenant],
+        autorefresh,
+    );
+
+    const {
+        Metrics = {},
+        PoolStats,
+        StateStats = [],
+        MemoryUsed,
+        Name,
+        CoresUsed,
+        StorageGroups,
+        StorageAllocatedSize,
+        Type,
+        SystemTablets,
+    } = tenant || {};
+
+    const tenantType = mapDatabaseTypeToDBName(Type);
+    const memoryRaw = MemoryUsed ?? Metrics.Memory;
+
+    const memory = (memoryRaw && bytesToGB(memoryRaw)) || 'no data';
+    const storage = (Metrics.Storage && bytesToGB(Metrics.Storage)) || 'no data';
+    const storageGroups = StorageGroups ?? 'no data';
+    const blobStorage = (StorageAllocatedSize && bytesToGB(StorageAllocatedSize)) || 'no data';
+    const storageEfficiency =
+        Metrics.Storage && StorageAllocatedSize
+            ? `${((parseInt(Metrics.Storage) * 100) / parseInt(StorageAllocatedSize)).toFixed(2)}%`
+            : 'no data';
+
+    const cpuRaw = CoresUsed !== undefined ? Number(CoresUsed) * 1_000_000 : Metrics.CPU;
+
+    const cpu = formatCPU(cpuRaw);
+
+    const metricsInfo = [
+        {label: 'Type', value: Type},
+        {label: 'Memory', value: memory},
+        {label: 'CPU', value: cpu},
+        {label: 'Tablet storage', value: storage},
+        {label: 'Storage groups', value: storageGroups},
+        {label: 'Blob storage', value: blobStorage},
+        {label: 'Storage efficiency', value: storageEfficiency},
+    ];
+
+    const tabletsInfo = StateStats.map((info) => {
+        if (info.VolatileState)
+            return {label: TABLET_STATES[info.VolatileState], value: info.Count};
+        return {};
+    });
+
+    const renderName = (tenant?: TTenant) => {
+        const {Name = TENANT_DEFAULT_TITLE, State} = tenant || {};
```

Comment:
You can use data that destructured on lines 50-61 (but introduce `TENANT_DEFAULT_TITLE` only here)
```
    const renderName = () => {
        return (
            <div className={b('tenant-name-wrapper')}>
                <EntityStatus
                    status={State}
                    name={Name || TENANT_DEFAULT_TITLE}
                    withLeftTrim
                    hasClipboardButton={tenant}
                    clipboardButtonAlwaysVisible
                />
            </div>
        );
    };
```